### DISCUSSION TITLE
how can we use  pg_partman for postgres on windows

### DISCUSSION THREAD

**Discussion by gr185136** [pgpartman/pg_partman#453] [Q&A]
I am not clear on how Pg_partman can be used on windows. not clear with adding pg_partman as an extension

Regards,
Gautham.R

**Comment by keithf4** [comment]
I apologize, but I do not do any development or maintenance of PostgreSQL on windows, so I cannot provide adequate guidance on this. However, it is no different than any other extension that would be managed on Windows, so if you can find other ones that work, it shouldn't be much different.

**Comment by patopat** [comment]
@gr185136 Did you find how to install this extension in windows ?
I’m on the same situation.

Regards,
Pat

**Comment by vipvdj** [comment]
Anyone found how to install this extension in windows ?
I’m on the same situation.

Regards,

**Comment by smw0718** [comment]
windows 如何安装呢

**Comment by smw0718** [comment]
也没有大佬说一下
